templates/vue/src/components/Lightbox/media/MultiContentMedia/index.vue

```diff
@@ -17,17 +17,44 @@
         <completed-icon :node="node" class="mx-2" />
       </h1>
     </header>
-    <multi-content-rows
-      v-if="node.presentationStyle"
-      :dimensions="dimensions"
-      :node="node"
-      :rowId="rowId"
-      :context="context"
-      :level="level"
-      @load="$emit('load')"
-      @change-row="changeRow"
-      @complete="complete"
-    />
+    <template v-if="node.presentationStyle">
+      <div v-if="node.presentationStyle === 'unit'">
+        {{
+          node.childOrdering.length
+            ? "The content of this unit has not been unlocked yet."
```

Comment:
Rather than this generic message, we should tell the user what they need to do to unlock the node (I think you can use the "locked-content" component for this).

Reply:
This text shows up only if the user opens an unlocked unit node with all its children locked and hidden (even if just one of the children is locked but not hidden, this text will not be shown and instead the locked-content prompt of that node will be shown). If we display the locked-content prompt, it will essentially let the user know that there are child nodes contained within the unit and let them see the conditions for locking the node, making one of the nodes "not hidden" when they should be. There are two issues here:

1. The behaviour would become the same as the "locked but not hidden" behaviour, since for locked nodes we will display the conditions. But for a hidden node, wouldn't it also be hidden when viewed from the parent, such that to the viewer the child would essentially not exist at all? So even just showing a locked-content prompt reveals some information that might not be what the author want when they decided to make the node hidden. (Also having all children of a multi-content node hidden while not hiding the multi-content node itself would be a strange decision on the author's side, so usually authors should just hide the multi-content node itself to achieve the desired result.)
2. If we do decide to use a locked-content prompt to let the user know what they need to do to unlock the node, which node's conditions should we display here? Since a unit can have multiple children that are all hidden, and each may have a different set of conditions, there may be some ambiguity as to which condition should be shown.

Reply:
After discussion, we will use the text for the "has no content" situation when all children are hidden so that they behave as if there are no children attached to the multi-content node.